mep/pages/models.py

```diff
@@ -6,14 +6,14 @@
 from django.template.defaultfilters import striptags, truncatechars_html
 from django.utils.functional import cached_property
 from django.utils.text import slugify
-from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
-from wagtail.core import blocks
-from wagtail.core.fields import RichTextField, StreamField
-from wagtail.core.models import Page
+from wagtail.admin.panels import FieldPanel
+from wagtail import blocks
+from wagtail.fields import RichTextField, StreamField
+from wagtail.models import Page
 from wagtail.documents.blocks import DocumentChooserBlock
 from wagtail.embeds.blocks import EmbedBlock
 from wagtail.images.blocks import ImageChooserBlock
-from wagtail.images.edit_handlers import ImageChooserPanel
+# from wagtail.images.edit_handlers import ImageChooserPanel
```

Comment:
you can remove instead of commenting out, we can always check previous versions in git history
```suggestion
```